opensciencegrid/s3-backup/build-config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "standard_build": true,
+    "repo_build": false,
+    "base_os": ["el7"],
+    "osg_series": ["3.6"],
+    "base_repo": ["development"]
+  }
```

Comment:
This PR should be limited to adding the framework by which we can change the matrix. If we decide that we want to change the set of tags that we build for this image, it should go in a different PR